src/dotnet/Chat.Service/Db/DbReaction.cs

```diff
@@ -0,0 +1,41 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace ActualChat.Chat.Db;
+
+[Table("Reactions")]
+[Index(nameof(ChatId), nameof(EntryId), nameof(AuthorId))]
+public class DbReaction : IHasId<string>
+{
+    public DbReaction() { }
+    public DbReaction(Reaction model) => UpdateFrom(model);
+
+    [Key] public string Id { get; set; } = null!;
+    public string AuthorId { get; set; } = "";
+    public string ChatId { get; set; } = "";
+    public long EntryId { get; set; }
+    public long Version { get; set; }
+    public string Emoji { get; set; } = "";
```

Comment:
Not sure if I saw this, but IMO you need a code that checks which reactions are allowed - i.e. it shouldn't be possible to send any random character as a reaction.